src/clang-c-frontend/padding.cpp

```diff
@@ -165,6 +165,9 @@ void add_padding(struct_typet &type, const namespacet &ns)
         it != components.end();
         it++)
     {
+      if(it->type().get_bool("#bitfield"))
+        assert(!it->type().width().empty());
+
```

Comment:
you may want to rewrite this as `assert(it->type().get_bool("#bitfield") && !it->type().width().empty())` or merge the assertion into the next `if`: compiling with `-DNDEBUG` will remove this assertion from the TU.

Reply:
Done, also moved some common sub-expressions to variables.